### DISCUSSION TITLE
de/2-06-kata-5/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#55] [General]
# de/2-06-kata-5/

https://ai.lernos.org/de/2-06-kata-5/

**Comment by ghost** [comment]
Vorschlag: Im Abschnitt "Lerne KI-Anwendungsfelder kennen" statt "nutze gerne die folgende Zusammenstellung möglicher KI-Anwendungsfelder im gleichnamigen Kapitel." besser schreiben: "nutze die Zusammenstellung möglicher KI-Anwendungsfelder im gleichnamigen Kapitel." und dabei KI-Anwendungsfelder als Link auf das gleichnamige Kapitel in den Grundlagen darstellen.

**Comment by tschaena** [comment]
"KI-Tools für Code (Unterstützung beim Prompting)": Müsste das nicht "KI-Tools für Code (Unterstützung beim Programmieren)" heissen?

**Comment by ThomasJenewein** [comment]
Bzgl. Use Case Analyse gibt es einen frei zugänglichen Toolkit des SAP Apphaus. Das ist in Workshop-Form oder auch verkürtzt durchführbar. incl. Templates wie ein Guide oder Mural Whiteboard 
hier der Toolkit: https://apphaus.sap.com/resource/generative-ai-explore-workshop  

Ich habe dazu auch einen Podcast mit Karen, der Autorin des Toolkits gemacht - FYI eine Zusammenfassung https://www.linkedin.com/pulse/innovating-sap-apphaus-dive-innovation-toolkit-its-new-jenewein-0duxe/  - 
eins meiner Learnings war die Use Case exploration iS des Human centered Design zu verstehen. dh z.B. die Design Thinking Kriterien und Ansätze zu nutzen, oder auch Tools wie eine Value Heatmap, um Use cases zu priorisieren (zB nach Value / Complexity). ich recherchiere gerne noch zum reduzierten Use-case Exploration Temmplate - das wäre ja evtl. eine gute Anleitung für eine KATA....

**Reply by olelai** [reply]
Wow - ein tolles Toolkit von SAP. Ich gehe es gerade alleine durch.

**Comment by rednaxelaesor** [comment]
Den Satz: "Um Dich bei dem gedanklichen Einstieg zu unterstützen, nutze gerne die folgende Zusammenstellung möglicher KI-Anwendungsfelder im gleichnamigen Kapitel." habe ich nicht gleich verstanden.
Da hier auf ein Kapitel in den Grundlagen verwiesen wird, könnte man das auch so schreiben. "... nutze gerne die ~folgende~ Zusammenstellung möglicher KI-Anwendungsfelder im **Grundlagenkapitel** "KI-Anwendungsfelder"
